src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx

```diff
@@ -339,10 +339,17 @@ export const RequestOverview = ({
     columnHelper.display({
       id: "actions",
       header: () => (
-        <Th borderBottom="1px solid" borderColor="gray.100" w="10rem" />
+        <Th
+          borderBottom="1px solid"
+          borderColor="gray.100"
+          w="10rem"
+          // NOTE: This line and the `display="none"` on the `HStack`
+          // should be removed once buttons are in on backend
+          display="none"
```

Comment:
actually do we need the `display=none` here? the gray background cuts off atm, we just want the display none on cells right

<img width="1399" alt="Screenshot 2023-03-29 at 6 29 06 PM" src="https://user-images.githubusercontent.com/22111124/228506318-7a6ee4f5-f07d-44c3-bba6-cd288aefd415.png">

Reply:
good catch, changed width to `0rem` to avoid this;  tfw u cant see good